index.html

```diff
@@ -17,8 +17,40 @@
 </head>
 <body>
   <header>
+    <h2>Code Journal</h2>
   </header>
   <main>
+    <div class="container">
+      <div data-view="entry-form" class="data-entry-form">
+        <form class="jornal-entry">
+          <h1>New Entry</h1>
+          <div class="row">
+            <div class="column-half column-full">
```

Comment:
these should just be set to column-half, that is why the layout class has that column `100%` width on mobile screens